src/textual/widget.py

```diff
@@ -82,6 +83,7 @@ def __init__(
         self._virtual_size = Size(0, 0)
         self._container_size = Size(0, 0)
         self._layout_required = False
+        self._default_layout = VerticalLayout()
```

Comment:
done while pair-programming with Will: it allows us to always have a layout for Widgets that have children, even if no layout was specified
(without this we have Widgets that don't display their children by default, which is a bit counter-intuitive)